jobbergate-api/jobbergate_api/tests/apps/applications/test_routers.py

```diff
@@ -14,6 +14,8 @@
 from jobbergate_api.apps.permissions import Permissions
 from jobbergate_api.storage import database, fetch_instance
 
+pytestmark = pytest.mark.usefixtures("startup_event_force")
```

Comment:
Please add a comment for these that explains why this is here.

Reply:
solved in [2c6e84c](https://github.com/omnivector-solutions/jobbergate/pull/212/commits/2c6e84cb5081f4457c9468553293adeea73d618f)